open_instruct/grpo_fast.py

```diff
@@ -323,6 +325,25 @@ class Args:
     eval_priority: Literal["low", "normal", "high", "urgent"] = "normal"
     """the priority of auto-launched evaluation jobs"""
 
+    # Tool settings
+    tools: Optional[List[str]] = None
+    """If set, use the tool mapped to the string. Currently only supports `search` and `code`"""
+    max_tool_calls: int = 5
```

Comment:
I guess one thing we could do to improve `max_tool_calls` is `max_tool_calls: List[int]`, so that it 1) either has len = 1, in which case, the same `max_tool_calls` applies to all the tools, or 2) it has the same length as tools, in which case each tool gets its own max_tool_calls.

Reply:
done!